vit-servicing-station-cli/src/csv/models.rs

```diff
@@ -140,7 +137,8 @@ impl From<Proposal> for proposals::Proposal {
             chain_vote_encryption_key: proposal.chain_vote_encryption_key,
             fund_id: proposal.fund_id,
             challenge_id: proposal.challenge_id,
-            challenge_type: proposal.challenge_type,
+            // this is hardcoded here, it is not used since the value will not be saved into DB
+            challenge_type: ChallengeType::Simple,
```

Comment:
Sadly we cannot, since it is the one used to serialize and be returned in the endpoints. It is the one that matches the aggregated proposal table. I understand that it may be better to have separated elements, but it would also be more cumbersome to maintain changes in all of them (we have a bunch of duplicated already).

Reply:
OK, as long as we never lie about the challenge type in the endpoints.